assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -0,0 +1,468 @@
+/**
+ * Analytics 4 test data mock.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import md5 from 'md5';
+import faker from 'faker';
+import invariant from 'invariant';
+import castArray from 'lodash/castArray';
+import { zip, from, Observable } from 'rxjs';
+import { map, reduce, take } from 'rxjs/operators';
+import isPlainObject from 'lodash/isPlainObject';
+
+/**
+ * Internal dependencies
+ */
+import { MODULES_ANALYTICS_4 } from '../datastore/constants';
+import { isValidDateString } from '../../../util';
+import { stringToDate } from '../../../util/date-range/string-to-date';
+
+const ANALYTICS_4_METRIC_TYPES = {
+	totalUsers: 'INTEGER',
+	newUsers: 'INTEGER',
+	sessions: 'INTEGER',
+	conversions: 'INTEGER',
+	screenPageViews: 'INTEGER',
+	engagedSessions: 'INTEGER',
+	averageSessionDuration: 'TIME',
```

Comment:
Hey @asvinb, it turns out we need to update the type names here seeing as they end up in the response, they should be as per this screen grab:

![image](https://user-images.githubusercontent.com/18395600/212956601-7452ef87-f41d-4b5d-b091-1d43f381871e.png)

Reply:
👍